threads/views.py

```
@@ -72,23 +71,64 @@ def get_reactions_count(self, request, *args, **kwargs):
 class CommentViewSet(viewsets.ModelViewSet):
     queryset = Comment.objects.all()
     serializer_class = CommentSerializer
+    permission_classes = [IsAuthenticated]
+
+    def get_thread(self):
+        try:
+            return Thread.objects.get(id=self.kwargs["threads_pk"])
+        except Thread.DoesNotExist:
+            raise PermissionDenied("Thread not found")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)